en/docs/install-and-setup/upgrading-the-solution/modifications-after-migration.md

```diff
@@ -0,0 +1,107 @@
+This page explains the modifications that need to be performed after the upgrade/migration process.
+
+## Open Banking Internal Scopes
+
+Configure open banking internal scopes as follows:
+
+1. Open the `<IS_HOME>/repository/conf/deployment.toml` file.
+
+2. Update the allowed scopes according the sample below: 
+
+      ``` toml
+      [oauth]
+      allowed_scopes = ["consent.*", "OB_.*"]
+      ```
+
+## Two-Factor Authentication
+
+If the migrated Service Provider is configured to use two-factor authentication:
+
+1. Update the `<IS_HOME>/repository/conf/common.auth.script.js` file with the following steps:
+ 
+       ``` 
+       var psuChannel = 'Online Banking';
+        
+       function onLoginRequest(context) {
+           publishAuthData(context, "AuthenticationAttempted", {'psuChannel': psuChannel});
+           doLogin(context);
+       }
+        
+       var doLogin = function(context) {
+           executeStep(1, {
+               onSuccess: function (context) {
+                   //identifier-first success
+               Log.info("Authentication Successful");
+                   publishAuthData(context, "AuthenticationSuccessful", {'psuChannel': psuChannel});
+                   OTPFlow(context);
+               },
+               onFail: function (context) { //identifier-first fail
+               Log.info("Authentication Failed");
+                   publishAuthData(context, "AuthenticationFailed", {'psuChannel': psuChannel});
+                   doLogin(context);
+               }
+           });
+       };
+        
+       var OTPFlow = function(context) {
+           executeStep(2, {
+                    //OTP-authentication
+               onSuccess: function (context) {
+                   context.selectedAcr = "urn:cds.au:cdr:2";
```

Comment:
This seems to be a CDS-specific common-auth script.

Reply:
Fixed in 5ab485a6f260c6a14c23e5001d46d98e02fb8e69